index.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Capstone project</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js" defer></script>
+</head>
+<body>
+    <section class="header-section">
+        <div class="main-containers">
+            <nav class="inner-header">
+                <li class="inner-list"><img src="images/facebook.png" alt=""></li>
+                <li class="inner-list"><img src="images/twitter.png" alt=""></li>
+                <li class="inner-list"><a href="#">English</a></li>
+                <li class="inner-list"><a href="#">My page</a></li>
+                <li class="inner-list"><a href="#">Logout</a></li>
+            </nav>
+        </div>
+            <header>
+                <div class="main-containers">
+                    <div class="header-container">
+                      <nav class="header">
+                        <div class="my-logo">
+                            <a href="#"><img class="logo" src="images/Ifeanyi_Logo_1B-01-removebg-preview.png" alt=""></a>
+                        </div>
+                        <div class="hambuger">
+                          <div class="bar"></div>
+                          <div class="bar"></div>
+                          <div class="bar"></div>
+                        </div>
+                        <ul class="nav-list">
+                          <li class="nav-item about"><a href="about.html">About</a></li>
+                          <li class="nav-item"><a href="#section1">Program</a></li>
+                          <li class="nav-item"><a href="#section2">join</a></li>
+                          <li class="nav-item"><a href="#section3">Sponsor</a></li>
+                          <li class="nav-item"><a href="#">News</a></li>
+                          <li class="nav-item"><a class="campaign" href="#">Campaign</a></li>
+                        </ul>
+                      </nav>
+                    </div>
+                </div>
+            </header>
+    </section>
+    <main>
+        <section class="title-section">
+            <div class="main-container">
+                <div class="page-title">
+                    <span class="greetings">"Hello! friends"</span>
+                    <h1 class="learning">Ifeanyi305 Virtual Learning in Schools: Pros and Cons.</h1>
+                    <div class="description">
+                        This is an online platform for education whereby students can study remotely 
+                        with the help of either thier mobile phones,computer or laptop 
+                        using the internet. join us.
+                    </div>
+                    <div class="time">
+                    <h1>2022.10.14(Thu) ~ 17(Fri)</h1>
```

Comment:
- [ ] Kindly change this to an `hx` like h2 h3 h4 as the `h1` tag is reserved for the main title of the page